README.md

```diff
@@ -0,0 +1,109 @@
+# HDF5 VOL Tests
+
+This repository contains several test applications that exercise [HDF5](https://github.com/HDFGroup/hdf5)'s
+public API and serve as regression tests for HDF5 [VOL Connectors](https://portal.hdfgroup.org/display/HDF5/Virtual+Object+Layer).
+
+### Software Requirements
+
+The following are required to build the HDF5 VOL tests:
+
+- CMake 2.8.12.2 or newer
+- HDF5 1.12.0 or newer
+
+Note: The HDF5 VOL tests project is currently unsupported for building on Windows.
+
+### Build Process
+
+The HDF5 VOL tests are built using CMake in an out-of-source fashion. A typical build
+would look like:
+
+    mkdir build
+    cd build
+    cmake -DCMAKE_BUILD_TYPE:STRING=Release ${BUILD_OPTIONS} -G ${CMAKE_GENERATOR} ..
+    make
+
+Build time options that can be passed for `${BUILD_OPTIONS}` are discussed in the following
+section. The CMake documentation should be consulted when determining what to set for
+`CMAKE_GENERATOR`.
+
+### Build Options
+
+The following build options are available to influence how the VOL tests get built. To set an
+option, it should be prepended with `-D` when passed to the `cmake` command. For example,
+
+    cmake -DHDF5_DIR=/temp/hdf5 ..
+
+would set the value of the `HDF5_DIR` build option to `/temp/hdf5`.
+
+##### Required
+
+`HDF5_DIR` - The VOL test build process will attempt to automatically locate an HDF5 installation
```

Comment:
If the build process can locate an HDF5 installation, why is `HDF5_DIR` a required option? Please put it under Optional or remove Optional subsection entirely.

Reply:
`HDF5_DIR` is required because the VOL tests cannot be built if an HDF5 installation is not available. An installation may be able to be located, but the user must provide this otherwise. It's not optional and it doesn't make sense to remove the optional subsection either because there are other options there that actually are optional.